### PR TITLE
New mapblips

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Updates map blips thanks to GregorMN.

See icondiff.

Edit: Diffbot is being useless again, here's a few examples.
image

## Why It's Good For The Game
Better art gud.
## Changelog
:cl:
imageadd: Updated minimap blips
/:cl:

### CODE REVIEW

**Comment by GrayRachnid** [comment]
image
guh?

**Comment by yyzsong** [comment]
image
...huh? I think you messed up adding these.

Also, I think this change is unnecessary. Our minimap sprites aren't the best, but they're simple and pretty decently easily readable, where the new sprites are less so (xeno caste specific ones are incomprehensible). I don't think we need shading for an icon that's the size of your fingernail, especially given that they're going to be constantly moving over the minimap. Shading also doesn't fit with the minimap itself, which itself is very barebones and only has solid colours, no shading.

I sincerely hope you did not pay for this.

**Review by Kuro020 — APPROVED** [review]

**Comment by Lumipharon** [comment]
>...huh? I think you messed up adding these.
> 

Icondiff is just being fucky for whatever reason, been happening for weeks.

**Comment by coldironwarrior** [comment]
the leader and engi blips threw me for a loop; the command team ones and all the large map blips look nice though.

**Comment by Wirpy** [comment]
They're so sexy, so cool. I love them.

**Comment by efzapa** [comment]
Cap and SO ones looks a bit jank smashed against the border

**Comment by Waselon** [comment]
overcomplicated and unreadable

**Comment by FosterGrime-Heymynameslouie** [comment]
this looks really good all of it

Except for the green and red crosses
they blend too much into the minimap I didnt even know they wree supposed to be green crosses till someone pointed it out

**Comment by gryffinp** [comment]
Pretty much universally less distinguishable, especially corpse-crosses.

image

Can't oppose enough.

**Comment by dragomagol** [comment]
image

This is pretty unreadable from a xeno point of view. They look good in the xeno menu that shows all of the active members, but the lack of a defining border makes them blend into each other and the map.

**Comment by yyzsong** [comment]
Why are all the disks just the same blue disk with different colour on what's supposed to be the white part?

**Comment by Gregormack** [comment]
image

The test was a complete success. Everything would be redone. Stay tuned! :)

**Comment by Gregormack** [comment]
> Why are all the disks just the same blue disk with different colour on what's supposed to be the white part?

Because it's a new design. Might change it though, didn't look as good as intended on the map

**Comment by Breadeagle** [comment]
These icons after playing a round are really hard to read. They blend into the background a lot, and I dislike the shading because I think it contributes. A border and keeping colours/shapes mostly solid would help a lot, but at that point the style is completely different. I can't really support this change.

**Comment by tmurphy7** [comment]
Before I get into criticisms, if I don't mention something here, it's because I liked it!

I don't really like the basic marine icon (the pistol) given it is different than the overhead icon (the star? asterisk?  whatever it is.), but that's my only complain for that icon.
The miners icons seem like just blurry versions of the previous icons, which weren't good before.
The disks predominantly use a similar hue, which makes them a little hard to distinguish at a glance.  The colors inside them are a bit washed out, as well.
The nuke icon needs a slight tweak.  It almost looks like an emoji.
Crosses weren't as distinguishable as before, which I think are one of the most important map icons.

I haven't played xenos since this TM so nothing to say on those.

**Comment by imsxz** [comment]
The phoron and plat miners look extremely similar. 

I don’t like that all the nuke disks are mostly blue icons, I wish they were colored like the old nuke disk icons. I like the spinning progress icons though.

I like the FC icon

I couldn’t tell which xenos were leaders on the minimap, they didn’t have the blue crown.

The minion icon looks kinda like a heart or something, doesn’t look like an angry face.

**Comment by FosterGrime-Heymynameslouie** [comment]
> image
> 
> The test was a complete success. Everything would be redone. Stay tuned! :)

hiii gregooor
can the green and red crosses be more green and red? like more contrast because right now they blend rather well with the minimap and the other blips, which, if you want to identify who and who isint dead and who to recover, might not be ideal

**Comment by Gregormack** [comment]
> > image
> > The test was a complete success. Everything would be redone. Stay tuned! :)
> 
> hiii gregooor can the green and red crosses be more green and red? like more contrast because right now they blend rather well with the minimap and the other blips, which, if you want to identify who and who isint dead and who to recover, might not be ideal

Will do buddy!

**Comment by jaggaaff** [comment]
These sprites are so chunky and awful to read. The minimalist style is the best choice on something like ss13 and especially on something like a minimap. They also don't seem to fit the style here. Why the fuck are we bothering to use this?

**Comment by DeltaFire15** [comment]
Looks horrible to read. High amount of coloration and shading makes them feel like neon lights. Many icons blend together visually creating an indistinguishable mess due to this approach of flashyness over readability.
These are icons for a *tactical* minimap. Primary concern should be being able to accurately discern things very quickly, not creating some pixelart masterpiece. Sometimes, simplicity rules.

**Comment by Nuker555** [comment]
Too complicated, didn't need to get changed in the first place.

**Comment by Breadeagle** [comment]
The more I play with it the more I find things to dislike, sadly. I've already commented a lot of the problems with it, but there's a few more. 

Xeno t4s dropped their crown in favour of a distinct sprite, but keep the gradients, making each one hard to tell from each other. T4s _share_ their sprite with xeno leaders now. Xeno leaders having a sprite overlaid onto their caste sprite in the past, now being replaced with a blue version of the t4 sprite. This makes it impossible to distinguish leaders on the map, which can be any caste. It also makes them inconsistent with hive status screen, which shows leaders original caste. 

Every sprite being more gradient and less "filler colour" makes everything have less of a distinguished border, and contributes to icons blending into each other, because there's less of a defined end point for each one. Since icons will often overlap each other, the blank space was honestly rather important to keep the identifying parts of their icon in view more often. 

Xeno readability is even further hampered by the fact that the background colour for each tier is not shared. They use a similar colour to their tier now, while in the past, every xeno icon had a purple background unless it was a structure detecting something. 

Minions now look like a purple version of t4 sprites at a glance.
The contrast of idle to active turrets is harder to see now - the central colour has far less contrast to the flashing colour compared to before. In fact its hard to tell if the central colour for turrets is even the same without a colour picker.
Digsites look way too similar to t1 xenos due to using a similar palette. Mostly for observers.

So in summary, readability of sprites is decreased overall by using more of the background space, broadening the palette, reducing consistency, and reducing contrast. In at least one case (xeno leaders), information is actively lost. If people think it actually looks better (I don't, personally), fine, but I feel like we should consider the purpose of map icons to begin with. Its a navigational and situational awareness tool. Readability should be preserved before aesthetic, because frustration will come from it being worse at its use, rather than it being (arguably) ugly.

**Comment by Deputi** [comment]
Sorry, these icons are not good for the game. The icons blend together on the minimap in a way that makes them very hard to discern from each other. I think this is due to the attempt of adding gradients to 7x7 pixel icons which just makes them look like blobs. Please get rid of the gradient, return the colour palette of the xeno icons, and dejank the miner icons, then these might be good icons.

**Comment by AlexCrimson** [comment]
Old icons looked sharper and as a result were much easier to read. Not a fan of these new ones. I understand that maybe its an attempt to add some more depth to the icons, but its just unneeded, and overall looks worse.

**Comment by Weedmaster77** [comment]
They hurt my eyes and are way to chunky seem to overlap worse the then old ones too. Only one that is new that I do like is the new one for the Disk think those are fine. But all the other ones are so hard to tell what they are other then the few that are just copy paste for the old ones.

**Comment by Windryn** [comment]
Love the art style, looks cute as hell.

But it needs work, the icons when in game look blurry and are hard to tell apart once the marines are clustered together.

Revival icons for the marines suffer the biggest imo.

**Comment by Kuro020** [comment]
most calm sprite discussion in tgmc

**Comment by yyzsong** [comment]
> most calm sprite discussion in tgmc

Unironically I think it might be

**Review by Kuro020 — CHANGES_REQUESTED** [review]
We'll wait for gregor to fix these if he wants

**Comment by coldironwarrior** [comment]
already commented but did just want to say that i really appreciate the differing turret colors this adds based on team! also into the more obvious OB and warning markers. spiderling icon is also a favorite of mine.

apologies if i missed it in the mosaic of all the icons; puppeteer puppets have a unique icon too; i don't think they're actually included in this? would be nice to see those be upgraded to the same quality as the little spiderling minion blip. unless you got to it and i just am not seeing it in icondiff

**Comment by Lumipharon** [comment]
will reopen if gregor reworks the icons

**Comment by Gregormack** [comment]
Holy shit, so many actually useful feedback. Thank you people, will try to improve. Apologies for bad results, but we'll get there, eventually!
